core-backend/src/error.rs

```diff
@@ -127,6 +127,8 @@ pub enum UnrecoverableExecutionError {
     TooBigReadLen,
     #[display(fmt = "Cannot take data in payload range from message with size")]
     ReadWrongRange,
+    #[display(fmt = "Unexpected version of execution settings encountered ")]
+    UnexpectedExecSettingsVersion,
```

Comment:
Unused error

Reply:
I think this is counted as usage, isn't it?
https://github.com/gear-tech/gear/blob/3cdd83aedafed5473d15d9b5c4b1a521dedff95d/core-processor/src/ext.rs#L736